scania-digital-design-system/tegel
# feat(grid): prefix change

**Describe pull-request**  
Changing prefix in Grid package

**Solving issue**  
Fixes: DTS-1614

**How to test**  
1. Go to the Storybook link below
2. Check if grid stories work correctly and if they have "tds" as a prefix.
3. Grid stories should behave as in [old prod Storybook](https://production.d1g8v8mrkx992n.amplifyapp.com/?path=/story/foundation-grid--basic)

**Suggested test steps**
